nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/pom.xml

```diff
@@ -72,7 +72,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
```

Comment:
@younggyuchun It looks like `guava` is not being used anywhere in this module. The dependency can be deleted.

Reply:
I just tested this out with the Mongo integration test suite and it doesn't appear to have any surprises for us, so LGTM too.